jdaviz/configs/mosviz/plugins/parsers.py

```diff
@@ -117,6 +117,7 @@ def mos_nirspec_directory_parser(app, data_obj, data_labels=None):
         # The amount of images needs to be equal to the amount of rows
         # of the other columns in the table
         if len(images) == len(spectra_1d):
+            images.sort()
```

Comment:
Why not sort it at glob time? `images = sorted([...])`

Reply:
Done